.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+    - master
+
+jobs:
+  checks:
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [ubuntu-latest, macos-latest]
```

Comment:
Can we add Windows? Or create an issue to add Windows later?

Reply:
scikit-build has issues with Windows, see https://github.com/scikit-build/scikit-build/pull/526 . Eventually, yes.

Reply:
cpp and pure work on Windows, the others didn't. Can try adding it back after this goes in.